lib/traject/met_csv_config.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require_relative 'csv_reader'
+require_relative 'dlme_json_resource_writer'
+require_relative 'macros/met_csv'
+require_relative 'macros/csv'
+extend Macros::MetCsv
+extend Macros::Csv
+settings do
+  provide 'writer_class_name', 'DlmeJsonResourceWriter'
+  provide 'reader_class_name', 'CsvReader'
+  provide 'allow_empty_fields', true
+end
+
+# MET Museum
+to_field 'id', column('Object ID')
+to_field 'cho_format', column('Object Name')
+to_field 'cho_creator', generate_creator
+to_field 'cho_spatial', column('City')
+to_field 'cho_has_type', column('Classification')
```

Comment:
just capturing logic here + will make sure is clear in mapping docs too:
cho_format == form ; 
cho_has_type == genre ; 
cho_edm_type == normalized DCMI Type like ; 
cho_type == not normalized DCMI Type like. 
In the Met CSV, none of those are handled well (is mostly forms / genres in the Getty AAT vein), so mainly capturing the first two fields and inferring the last two.

I'm still trying to confirm the usage of 3D objects in places like MODS Types and relation to something like 3D objects in places like the EDM usage, and I think this may require a fix, but we can update that with a PR to not block this.